ci/cron/src/Main.hs

```diff
@@ -245,13 +241,11 @@ fetch_gh_versions pred = do
 fetch_s3_versions :: DocOptions -> IO Versions
 fetch_s3_versions opts = do
     -- On the first run, this will fail so treat it like an empty file.
-    -- We could technically remove the catch later.
```

Comment:
Keeping this on the off chance we'd need to reinitialize the docs bucket from scratch some day.